react/DateMonthPicker/index.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import Button from '../Button'
+import Icon from '../Icon'
+import { translate } from '../I18n'
+import range from 'lodash/range'
+import { format } from 'date-fns'
+import styles from './styles.styl'
+import cx from 'classnames'
+
+const MonthButton = translate()(({ monthNum, f, onClick, isSelected }) => {
+  const d = new Date(2019, monthNum, 15)
```

Comment:
Are year and day hard coded here because we don't care about it in this component? Maybe we should add a comment?

Reply:
Done, thanks